{"message":"Unable to ensure uniqueness for usernames: ","name":"MongoError","stack":"MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","level":"warn","timestamp":"2016-10-18T09:45:29.825Z"}
{"message":"Unable to ensure uniqueness for user email addresses: ","name":"MongoError","stack":"MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","level":"warn","timestamp":"2016-10-18T09:45:29.829Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:40.269Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:40.279Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:40.289Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:40.289Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:41.398Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:41.400Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:41.403Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:41.406Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:42.473Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:42.473Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:42.473Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:42.473Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:42.473Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:42.489Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:43.609Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:43.609Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:43.609Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:43.609Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:43.640Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:43.640Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:44.876Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:44.876Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:44.876Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:44.876Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:44.892Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:44.892Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:46.396Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:46.396Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:47.584Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:47.584Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:51.265Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:51.269Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:52.541Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:52.541Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:53.584Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:53.584Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:55.426Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:55.426Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [localhost:27017] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' }","timestamp":"2016-10-18T09:45:58.338Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [localhost:27017] on first connect' } MongoError: failed to connect to server [localhost:27017] on first connect\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at emit (events.js:188:7)\n    at .<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (E:\\Node\\langrenserver\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:286:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-18T09:45:58.338Z"}
